examples/lock-app/cc32xx/main/debug_settings.h

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+//*****************************************************************************
+// Includes
+//*****************************************************************************
+// Standard includes
+#ifndef DEBUG_SETTINGS_H
```

Comment:
nit: chip standard is generally using `#pragma once`

consider deleting the `includes` and `Standard includes` comments as they do not seem to help reading the code.